src/components/activities/ActivityCard.jsx

```diff
@@ -7,6 +7,13 @@ import PropType from 'prop-types';
  * @extends React.Component
 */
 class ActivityCard extends Component {
+  static propTypes = {
+    category: PropType.string.isRequired,
+    date: PropType.string.isRequired,
+    description: PropType.string.isRequired,
+    points: PropType.string.isRequired,
+    status: PropType.string.isRequired,
+  };
   constructor(props) {
```

Comment:
For this component we do not yet need the constructor coz there is no state we need to initialize

Reply:
Thanks for the pointer @joykare  remove ill remove this constructor